CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 Next Release
 -------------
+1.4.0
```

Comment:
remove the 1.4.0 and put this directl under `Next Release`. Who ever cuts the release will update the changelog version.